spec/api-protocol-spec.js

```diff
@@ -394,7 +396,7 @@ describe('protocol module', function () {
       var handler = function (request, callback) {
         callback(fakeFilePath)
       }
-      protocol.registerBufferProtocol(protocolName, handler, function (error) {
+      protocol.registerFileProtocol(protocolName, handler, function (error) {
```

Comment:
not related to the PR, the specs were inside `protocol.registerFileProtocol`, so thought they are supposed to test it ?